src/components/PostsList.tsx

```diff
@@ -1,54 +1,53 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
It's not recommended to disable eslint rules unless absolutely necessary. The rule 'jsx-a11y/control-has-associated-label' ensures that interactive elements have an accessible name. It would be better to fix the issue instead of disabling the rule. If you're having trouble, the eslint-plugin-jsx-a11y documentation could be helpful.